### PR TITLE
release: add 3.0.7 release notes

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

add 3.0.7 release notes

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/pull/1687

### Which version does your change affect? 

dev, 3.1, 3.0

### CODE REVIEW

File: dev/releases/3.0.7.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+发版日期：2019 年 12 月 4 日
+
+TiDB 版本：3.0.7
+
+TiDB Ansible 版本：3.0.7
+
+## TiDB
+
+- 修复 TiDB Server 本地时间落后于 TSO 时间时，可能造成锁的 TTL 过大的问题 [#13868](https://github.com/pingcap/tidb/pull/13868)
```
Comment (lilin90) [inline dev/releases/3.0.7.md]:
```suggestion
- 修复 TiDB server 本地时间落后于 TSO 时间时，可能造成锁的 TTL 过大的问题 [#13868](https://github.com/pingcap/tidb/pull/13868)
```

File: dev/releases/3.0.7.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- 修复从字符串解析日期时，由于使用本地时区（`gotime.Local`）而导致解析结果的时区不正确的问题 [#13793](https://github.com/pingcap/tidb/pull/13793)
```
Comment (lilin90) [inline dev/releases/3.0.7.md]:
```suggestion
- 修复从字符串解析日期时，由于使用本地时区 (`gotime.Local`) 而导致解析结果的时区不正确的问题 [#13793](https://github.com/pingcap/tidb/pull/13793)
```

File: dev/releases/3.0.7.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- 修复 `builtinIntervalRealSig` 的实现中，`binSearch` 方法不会返回 error，导致最终结果可能不正确的问题 [#13767](https://github.com/pingcap/tidb/pull/13767)
+- 修复整型数据被转换为无符号浮点/Decimal 类型时，精度可能丢失造成数据错误的问题 [#13755](https://github.com/pingcap/tidb/pull/13755)
+- 修复 Natural Outer Join 和 Outer Join 使用 “USING” 语法时，`not null` 标记没有被重置导致结果错误的问题 [#13739](https://github.com/pingcap/tidb/pull/13739)
```
Comment (lilin90) [inline dev/releases/3.0.7.md]:
```suggestion
- 修复 Natural Outer Join 和 Outer Join 使用 `USING` 语法时，`not null` 标记没有被重置导致结果错误的问题 [#13739](https://github.com/pingcap/tidb/pull/13739)
```

File: dev/releases/3.0.7.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+- 修复更新统计信息时可能存在数据竞争，导致统计信息不准确的问题 [#13687](https://github.com/pingcap/tidb/pull/13687)
+
+## TiKV
+
+- 判断死锁检测服务的第一个 region 时，加上 region 合法检测，防止信息不完整的 region 导致误判 [#6110](https://github.com/tikv/tikv/pull/6110)
```
Comment (lilin90) [inline dev/releases/3.0.7.md]:
```suggestion
- 判断死锁检测服务的第一个 Region 时，加上 Region 合法检测，防止信息不完整的 Region 导致误判 [#6110](https://github.com/tikv/tikv/pull/6110)
```

**Comment by yikeke** [comment]
Updated. PTAL again @lilin90

**Pull request merged**